docker-compose.yml

```diff
@@ -8,11 +8,21 @@ services:
     restart: always
     environment:
       MYSQL_DATABASE: puresynth
-      MYSQL_USER: ${DB_USERNAME}
-      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_USER: puresynth
```

Comment:
In both PRs (https://github.com/gdesiato/pure-synth/pull/16/) you're changing the docker-compose file.